carbon-design-system/carbon-addons-iot-react
# fix(TableCard): use table loading state rather than card

Closes #1904

**Summary**

- When `isLoading` is set in TableCard, set `view: { table: { loadingState: isLoading } }` instead of passing it to Card
- Also fixes a bug where `loadingState.rowCount` was not being respected

**Change List (commits, features, bugs, etc)**

- Add `isLoading` knob to all TableCard stories
- Add `isLoading` story to TableCard

**Acceptance Test (how to verify the PR)**

- Go to ?path=/story/watson-iot-tablecard--with-is-loading and see that the table loading state is being used and there are 7 rows
- Go to  /?path=/story/watson-iot-table--with-loading-state and see that 7 rows are shown
